content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -202,21 +203,21 @@ inputStreamReader = new InputStreamReader(socket.getInputStream());
 outputStreamWriter = new OutputStreamWriter(socket.getOutputStream());
 ``` 
 
-The Scanner class is used to get input from the user from the console/terminal. System. in is a keyboard input connected input stream system.
+The Scanner class is used to get input from the user from the console/terminal. System.in is a keyboard input connected input stream system.
 
 ```java
 Scanner scanner = new Scanner(System.in);
 ```
 
 ### Methods used in Client Socket
-1. `public InputStream( getInputStream())` - returns an InputStream object for receiving data. Our Server will keep on receiving messages until client sends "BYE".
-2. `public OutputStream( getOutputStream())`- returns an OutputStream for sending data.
-3. `BufferReader( )` - Wraps inputStreamReader to improve efficiency
-4. `BufferWriter( )` - Wraps outputStreamReader to improve efficiency
-5. `BufferedWriter.flush( )` - Flushes the streams...The flush method is invoked when the buffer is complete.
-6.  `void close() `- closes all streams and the socket.
+1. `public InputStream getInputStream()` - returns an InputStream object for receiving data. Our Server will keep on receiving messages until client sends "BYE".
+2. `public OutputStream getOutputStream()`- returns an OutputStream for sending data.
+3. `BufferReader()` - Wraps inputStreamReader to improve efficiency
+4. `BufferWriter()` - Wraps outputStreamReader to improve efficiency
+5. `BufferedWriter.flush()` - Flushes the streams...The flush method is invoked when the buffer is complete.
+6. `void close()`- closes all streams and the socket.
   
-### THIS IS A CLIENT PROGRAM THAT WILL BE COMMUNICATING WITH THE SERVER.
+### Full Client-Side Application
```

Comment:
Change: 

> ### Full Client-Side Application

To:

> #### Full Client-Side Application

Fixing heading structure since this is under: **Programming Client-Side**